src/analyses/goto_check.cpp

```diff
@@ -523,22 +584,44 @@ void goto_checkt::conversion_check(
   }
 }
 
+/// Check if the frontend has flagged a given expression for overflow
+/// checking.
+/// \param e: The expression that is to be checked.
+/// \return: True or false based on whether the flag was set or not.
+static bool overflow_check_flag_set(const exprt &e)
+{
+  return e.get_bool("overflow_check");
+}
+
```

Comment:
Same here -- introduce an expression.

Reply:
This is being addressed in another conversation for a similar issue - see above.